src/main/java/eu/cqse/teamscale/jenkins/upload/TeamscaleUploadBuilder.java

```diff
@@ -197,6 +197,10 @@ public void perform(@Nonnull Run<?, ?> run, FilePath workspace, @Nonnull Launche
         List<FilePath> files = new ArrayList<>();
 
         workspace.act(new FilePath.FileCallable<FilePath[]>() {
+
+            private static final long serialVersionUID = 1L;
```

Comment:
Are you sure that this fixes the problem? AFAIK `serialVersionUID` is optional for serialization. Does the object implement Serializable? Should be easy to try this out in a unit test.